crm_profiling/__manifest__.py

```diff
@@ -19,7 +19,7 @@
     """,
```

Comment:
@nikunjantala  @jans23 

I am not able to understand the line mentioned in "Summary"; 


**Note:** this module is not compatible with the module segmentation, since
it's the same which has been renamed.


Can you please elaborate a bit? Does this mean that we are trying to say that the module is not an extension but a replacement for the earlier segmentation module?

Reply:
Which other module do you mean? We maintain only whit one `crm_profiling`. Do you refer to another base module?